src/app/app-routing.module.ts

```diff
@@ -4,18 +4,29 @@ import { Routes, RouterModule, NoPreloading } from '@angular/router';
 import { AuthService } from './auth/service/auth.service';
 import { GuardService } from './auth/service/guard.service';
 import { UserGuardService } from './auth/service/user.guard.service';
+import { LoggedInCompanyGuardServiceService } from './auth/service/logged.in.company.guard.service';
 
 
 const routes: Routes = [
     {
         path: 'todo',
         loadChildren: () => import('./todo/todo.module').then(m => m.TodoModule),
-        canActivate: [GuardService]
+        canActivate: [LoggedInCompanyGuardServiceService]
     },
     {
         path: 'users',
         loadChildren: () => import('./users/users.module').then(m => m.UsersModule),
         canActivate: [UserGuardService]
+    },
+    {
+        path: 'company',
+        loadChildren: () => import('./company/company.module').then(m => m.CompanyModule),
```

Comment:
CompanyModule inicializuje store pre slice Company state. Tým že je tento modul lazy-loaded, ak idem po štarte appky rovno napr. na routu todo, v store neexistujú informácie o Company. Takže ani guarda ktorá overuje company v store nebude fungovať dobre... Pozor na to! Ak sú informácie o company dôležité pre beh appky, musia byť inicializované na začiatku, nie v lazy-loaded module.